content/en/docs/concepts/storage/projected-volumes.md

```diff
@@ -0,0 +1,202 @@
+---
+reviewers:
+- sftim
+- marosset
+- jsturtevant
+- zshihang
+title: Projected Volumes
+content_type: concept
+---
+
+<!-- overview -->
+
+This document describes the current state of _projected volumes_ in Kubernetes. Familiarity with [volumes](/docs/concepts/storage/volumes/) is suggested.
+
+<!-- body -->
+
+## Introduction
+
+A `projected` volume maps several existing volume sources into the same directory.
+
+Currently, the following types of volume sources can be projected:
+
+* [`secret`](/docs/concepts/storage/volumes/#secret)
+* [`downwardAPI`](/docs/concepts/storage/volumes/#downwardapi)
+* [`configMap`](/docs/concepts/storage/volumes/#configmap)
+* `serviceAccountToken`
+
+All sources are required to be in the same namespace as the Pod. For more details,
+see the [all-in-one volume design document](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/all-in-one-volume.md).
+
+### Example configuration with a secret, a downwardAPI, and a configMap {#example-configuration-secret-downwardapi-configmap}
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: volume-test
+spec:
+  containers:
+  - name: container-test
+    image: busybox
+    volumeMounts:
+    - name: all-in-one
+      mountPath: "/projected-volume"
+      readOnly: true
+  volumes:
+  - name: all-in-one
+    projected:
+      sources:
+      - secret:
+          name: mysecret
+          items:
+            - key: username
+              path: my-group/my-username
+      - downwardAPI:
+          items:
+            - path: "labels"
+              fieldRef:
+                fieldPath: metadata.labels
+            - path: "cpu_limit"
+              resourceFieldRef:
+                containerName: container-test
+                resource: limits.cpu
+      - configMap:
+          name: myconfigmap
+          items:
+            - key: config
+              path: my-group/my-config
+```
```

Comment:
I'd suggest we move this and the following YAML files out of the page, into the 'examples' directory.

Reply:
Or somewhere under tasks?

Reply:
OK, I will move the text into [configure-projected-volume-storage.md](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/configure-pod-container/configure-projected-volume-storage.md) with yaml in the [storage](https://github.com/kubernetes/website/tree/main/content/en/examples/pods/storage) examples folder.

Reply:
Looks like tasks are more like for trying out rather than just for explanations. I will just move the yaml to the examples folders and link to it.

Reply:
You can use a `codenew` shortcode rather than a link, if that's relevant / useful.